README.md

```diff
@@ -114,3 +114,45 @@ The interesting fields for this use case are:
   the SSO user ID to be sent to Segment
 - `status.compliantUsername` - Contains the username used in the cluster audit
   log.
+
+## Setting up a containerized Splunk instance
+- Either podman or docker can be user
+- running the following command will deploy a Splunk enterprise container:  
+- `podman pull docker.io/splunk/splunk:latest`
```

Comment:
No need to pre-pull, podman run can pull already.

Reply:
removed